File: desktop-editor/src/ru/myitschool/dcrawler/desktop/DesktopLauncher.java
package ru.myitschool.dcrawler.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import ru.myitschool.dcrawler.LevelManager;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
        config.height = 680;
        config.width = 1200;
        config.resizable = false;
		config.samples=10;
		new LwjglApplication(new LevelManager(), config);
	}
}

File: desktop-editor/src/com/wolg_vlad/dcrawler/desktop/DesktopLauncher.kt
package com.wolg_vlad.dcrawler.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.wolg_vlad.dcrawler.LevelManager

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.width = 1280
        config.height = 720
        LwjglApplication(LevelManager(), config)
    }
}